bugzilla_mcp/utils/bugzilla.py

```
@@ -58,7 +60,818 @@ async def add_comment(
+    async def bugs_comments(self, bug_ids: list[int]) -> dict[str, list[dict[str, Any]]]:
+        """fetch comments for multiple bugs in parallel using asyncio.gather"""
+        if not bug_ids:
+            return {}
+
+        import asyncio
+
+        async def fetch_one(bug_id: int):
+            try:
+                comments = await self.bug_comments(bug_id)
+                return str(bug_id), comments
+            except Exception:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)